### PR TITLE
Add upload image to contact

### PR DESCRIPTION
**Pull request by jadmsaadaot**
*Issue #:*
https://github.com/bcgov/met-public/issues/785

*Description of changes:*

- Add image upload functionality for contact Avatar
- Display saved image in public page
- Fix sorting of widget items

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by djnunez-aot — APPROVED** [review]

File: met-api/migrations/versions/03ee1815f6a6_.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# revision identifiers, used by Alembic.
+revision = '03ee1815f6a6'
+down_revision = '9536f547cdd5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('contact', sa.Column('avatar_filename', sa.String(), nullable=True))
```
Comment (saravanpa-aot) [inline met-api/migrations/versions/03ee1815f6a6_.py:21]: are we planning to just store the name or the full url?

File: met-api/src/met_api/services/contact_service.py [hunk 2]
```diff
@@ -11,13 +12,20 @@ class ContactService:
     def get_contact_by_id(contact_id):
         """Get contact by id."""
         contact_record = Contact.get_contact_by_id(contact_id)
-        return ContactSchema().dump(contact_record)
+        contact = ContactSchema().dump(contact_record)
+        contact['avatar_url'] = ObjectStorageService.get_url(contact.get('avatar_filename', None))
```
Comment (saravanpa-aot) [inline met-api/src/met_api/services/contact_service.py:16]: i forgot whats in the O.get_url() does ;just thinking will it be faster if we store the full url in the DB and avoid this logic? We can discuss this and weigh each approach

Comment (jadmsaadaot) [reply met-api/src/met_api/services/contact_service.py:16]: We can discuss this. The reasoning behind it is that this will work even if host changed.

**Review by saravanpa-aot — APPROVED** [review]

File: met-api/migrations/versions/03ee1815f6a6_.py (hunk 1 above)
Comment (jadmsaadaot) [reply met-api/migrations/versions/03ee1815f6a6_.py:21]: As agreed in our chat. We will discuss this with @tiago-aot in a meeting

**Pull request merged**